.circleci/config.yml

```diff
@@ -6,8 +6,48 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Build and Test
+          name: Prepare
+          command: npm install
+      - run:
+          name: Build
+          command: npm run build
+      - run:
+          name: Unit Test (with coverage)
+          command: npm run coverage
+      - run:
+          name: Prepare demo
+          command: ./scripts/demo.sh
+      - persist_to_workspace:
+          root: .
+          paths: ./demo
+  update-demo:
+    docker:
+      - image: node:8.10.0
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - add_ssh_keys:
+          fingerprints:
+              - "c3:03:93:79:5d:c7:23:b8:4a:4d:5d:41:e3:38:3f:84"
```

Comment:
Trying to understand more about this fingerprint..

https://superuser.com/questions/421997/what-is-a-ssh-key-fingerprint-and-how-is-it-generated
https://circleci.com/docs/2.0/configuration-reference/#filters

So it's meant to stay public on master?

Reply:
ya I also have the same problem and who owns this fingerprint?

Reply:
There's a key that allows circleci to push to github. The private part is own by circleci and the public part is on github. The fingerprint is just to identify which key to use. This is not sensitive.